pyproject.toml

```diff
@@ -36,36 +36,33 @@ lines_between_types = 1
 
 [tool.towncrier]
 package = "plasmapy"
+name = "PlasmaPy"
 filename = "CHANGELOG.rst"
 directory = "changelog/"
-issue_format = "`#{issue} <https://github.com/plasmapy/plasmapy/pull/{issue}>`__"
+wrap = true
```

Comment:
What is this?  I don't see it anywhere in the `towncrier` documentation?

Reply:
I found it under [further options](https://towncrier.readthedocs.io/en/latest/index.html?highlight=wrap#further-options), to limit lines to 79 characters.

I'm giving it a try in #1626 to see if it works.